rosvenv.bash

```diff
@@ -67,7 +77,15 @@ activateROS() {
         fi
 
         source "${ws_dir}/devel/setup.bash"
-        source "${ws_dir}/pyenv/bin/activate"
+        if test -f "$ws_dir/$CONDA_ENV_FILE_NAME"; then
+            echo "Found conda env. Sourcing"
+            CONDA_ENV_NAME=$(cat "$ws_dir/$CONDA_ENV_FILE_NAME")
+            conda activate $CONDA_ENV_NAME
+        else
+            echo "No conda env found. Sourcing venv"
+            source "${ws_dir}/pyenv/bin/activate"
+        fi
+
```

Comment:
Do we actually need the `echo` here? Should the change in `PS1` not be enough to inform the user about the sourced env, or do conda and venv make the same change?